Svc/CmdDispatcher/CommandDispatcherImpl.cpp

```diff
@@ -188,4 +188,20 @@ void CommandDispatcherImpl::pingIn_handler(FwIndexType portNum, U32 key) {
     this->pingOut_out(0, key);
 }
 
+void CommandDispatcherImpl::seqCmdBuff_overflowHook(FwIndexType portNum, Fw::ComBuffer& data, U32 context){
+    // Extract command opcode
+    Fw::CmdPacket cmdPkt;
+    Fw::SerializeStatus stat = cmdPkt.deserializeFrom(data);
+    FwOpcodeType opcode = 0; // Note: 0 = Reserved opcode    
+
+    if (stat == Fw::FW_SERIALIZE_OK){
+        opcode = cmdPkt.getOpCode();
+    }
+    
+    // Log Cmd Buffer Overflow and increment CommandsDroppedBufOverflow counter
+    this->m_numCmdsDropped++;
+    this->log_WARNING_HI_CommandDroppedQueueOverflow(opcode,context);
+    this->tlmWrite_CommandsDropped(this->m_numCmdsDropped);
```

Comment:
Two more things:
1. We should report tlmWrite_CommandsDropped on a sched call to prevent a tlm flood in response to a cmd flood
2. We should  add CommandDroped telemetry to `Ref/Top/RefPackets.fppi` this will   prevent the CI failure.

Happy to clean these nits up if you'd like help.